lib/modules/manager/poetry/extract.spec.ts

```diff
@@ -177,6 +178,30 @@ describe('modules/manager/poetry/extract', () => {
       });
     });
 
+    it('parses github dependencies tags on ssh urls', async () => {
+      const content =
+        '[tool.poetry.dependencies]\r\nfastapi = {git = "git@github.com:tiangolo/fastapi.git", tag="1.2.3"}\r\nwerkzeug = ">=0.14"';
```

Comment:
Using `codeblock` syntax is the preferred way now - easier to read and maintain

Reply:
Did I understand the suggestion correctly? (https://github.com/renovatebot/renovate/pull/21949/commits/81364455785463b63664d62c9c925a83b33e2205)
If so, would you like me to refactor remaining lines in that test file too? (Don't hesitate, I'd be glad to do so)

Reply:
It would be great if you could, thanks. It makes it a little harder for us to verify that no tests have "changed" (logically) but would be worth it. FYI @viceice

Reply:
I did it at https://github.com/renovatebot/renovate/pull/21949/commits/508e83cecaaf99b654b2d40fd4c3a1a9d12d2493
and will revert the commit if it makes it harder to review.